test/e2e/vault_test.go

```diff
@@ -0,0 +1,78 @@
+// +build vault
+
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/secrets-store-csi-driver/test/e2e/framework"
+	"sigs.k8s.io/secrets-store-csi-driver/test/e2e/framework/csidriver"
+	"sigs.k8s.io/secrets-store-csi-driver/test/e2e/framework/vault"
+)
+
+var _ = Describe("Testing CSI Driver with Vault provider", func() {
+	ctx := context.TODO()
+	csiNamespace := "secrets-store-csi-driver"
+
+	It("Install CSI driver and Vault provider", func() {
```

Comment:
This seems like it should go in a setup function instead of being in a test.

Reply:
Moved only CSI driver deployment to setup function